Google.Api.Generator.Rest/Models/ParameterModel.cs

```diff
@@ -91,6 +97,21 @@ private PropertyDeclarationSyntax GenerateProperty(SourceFileContext ctx)
                 .WithAttribute(ctx.Type<RequestParameterAttribute>())(Name, locationExpression);
             if (_schema.Description is string description)
             {
+                // TODO: Add a remarks section for repeated enum parameters.
+                property = property.WithXmlDoc(XmlDoc.Summary(description));
+            }
+            return property;
+        }
+
+        private PropertyDeclarationSyntax GenerateRepeatedProperty(SourceFileContext ctx)
```

Comment:
Wondering if you considered modifying `SchemaTypes.GetTypFromSchema`, so that it returns the `Repeatable<TEnum>` instead of the `TEnum`, and the exception is made here for the single property instead of the repeated one. Just maybe it's a little more realistic as to what is the real supported scenario, and what is the one we are supporting for compatibility.

I'm fine either way though, whatever has the less ramifications works best.

Reply:
Possibly - I'd have to investigate what that does elsewhere, e.g. in resources. It may well be that it just doesn't come up.